src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2648,6 +2647,12 @@ Generate\ summaries\ for\ entries\ in\ the\ group=Generate summaries for entries
 Generating\ summaries\ for\ %0=Generating summaries for %0
 Ingestion\ started\ for\ group\ "%0".=Ingestion started for group "%0".
 Summarization\ started\ for\ group\ "%0".=Summarization started for group "%0".
+Reset\ templates\ to\ default=Reset templates to default
+Summarization\ chunk=Summarization chunk
```

Comment:
This is the text the AI should complete. Thus, it should be named as that:

    Completion text for summarization of a chunk

I think "Summarization chunk" is not a term known by the community.

Simlar for the next

    Completion text for summarization of chunks